packages/@okta/vuepress-theme-prose/assets/css/components/_homePage.scss

```diff
@@ -6,10 +6,11 @@ $oktaGameButtonColor: #ffdd53;
   font-family: $font-family;
   min-height: 100vh;
   color: white;
-  overflow: hidden;
-  &--col-no-padding.col-lg-2,
-  &--col-no-padding.col-xl-2 {
-    padding: 0px 0px;
+  .width-adjuster.col-xl-10.col-lg-10.col-md-12.col-sm-12 {
```

Comment:
Is the rest of the selector here `.col-xl-10.col-lg-10.col-md-12.col-sm-12` necessary? `.width-adjuster` seems unique.

Reply:
I need a lot of weight in this selector to combine `-cols ` and own style for this to work. It's a dirty hack, but it's needed till some redesign changes in widths will came.